docs/requirements.txt

```diff
@@ -1 +1 @@
-.[torch,docs]
+.[torch,ray,docs]
```

Comment:
isn't ray automatically installed with the base deps?

Reply:
Yes 😌 Removed this